### PR TITLE
Fixed #29943 -- Doc'd that admin changelist may add pk to ordering.

### PR DESCRIPTION
**Pull request by hramezani**
[ticket_29943](https://code.djangoproject.com/ticket/29943)

### CODE REVIEW

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -1043,6 +1043,20 @@ subclass::
...
+
+          from django.db import models
+
+          class Author(models.Model):
+              name = models.CharField(max_length=100)
+
+              class Meta:
+                  ordering = ['name']
+
+        The objects in admin changelist are sorted by ``name`` and ``pk``.
```
Comment (charettes) [inline docs/ref/contrib/admin/index.txt]: Missing new line and we should probably mention _why_ the admin does that.

Comment (hramezani) [reply docs/ref/contrib/admin/index.txt]: Thank you for the comment. New line added also the reason added from
https://github.com/django/django/blob/6275b50ac26ea6c026aa7b2aae9192f5792e8a94/django/contrib/admin/views/main.py#L306

**Comment by hramezani** [comment]
@carltongibson thank you for the comment. the comment from the mentioned ticket added to note.
I also checked the override `get_ordering()`, so the behavior doesn't change.

File: docs/ref/contrib/admin/index.txt [hunk 2]
```diff
@@ -1037,6 +1037,33 @@ subclass::
...
+        It could have serious performance issues is if you have:
+
+        * a lot of rows
+        * an index on the sort_col
```
Comment (timgraham) [inline docs/ref/contrib/admin/index.txt]: What does `sort_col` refer to?

Comment (hramezani) [reply docs/ref/contrib/admin/index.txt]: The column which we want to sort result based on it. I got it from [17198#comment:2](https://code.djangoproject.com/ticket/17198#comment:2).

**Comment by timgraham** [comment]
Is this version sufficient?
```
.. admonition:: Performance considerations with ordering and sorting

    To ensure a deterministic ordering of results, the changelist adds
    ``pk`` to the ordering if it can't find a single or unique together set
    of fields that provide total ordering.

    For example, if you have a model with ``Meta.ordering = ['name']``
    (where ``name`` isn't unique), then the changelist is sorted by
    ``name`` and ``pk``. This could perform poorly if you have a lot of
    rows and don't have an index on ``name`` and ``pk``.
```
The other question is why is the note in `ModelAdmin.ordering`? I guess the same consideration from `Meta.ordering` apply to `ModelAdmin.ordering`.

**Comment by hramezani** [comment]
@timgraham thank you for the comment.
I replaced your note. it seems better. and moved them to `Meta.ordering`

**Comment by timgraham** [comment]
I didn't mean to suggest that. Documentation for contrib apps like admin doesn't belong in "core" documentation. if we had some other ideas about performance related admin concerns, perhaps a new section or document in ref/admin could be created.

**Comment by hramezani** [comment]
I am confused :)). I don't know what should I do :))
If this PR doesn't make sense it is better to close it.
Thank you again for the reply

**Pull request merged**